### ISSUE TITLE
spawnsub console command should properly generate wrecks

### ISSUE THREAD

**Issue by TypeStein** [Regalis11/Barotrauma#3325]
- [Yes] I have searched the issue tracker to check if the issue has already been reported.

**Description**
Currently spawnsub does not take into account if a sub is a wreck and spawns wrecks as subs without generating the thalamus.

**Steps To Reproduce**
spawn any wreck with the spawnsub command.

**Version**
stable and unstable.

**Comment by itchyOwl** [comment]
Implemented in Regalis11/Barotrauma-development@c02f104. To be honest, I didn't even know that we had such a command when I implemented the wrecks.

**Comment by TypeStein** [comment]
Awesome, does the spawnsub wreck get a dice roll to decide if it's a thalamus wreck or is it always a thalamus wreck?

**Comment by itchyOwl** [comment]
I added an extra parameter, like so: spawnsub Wreck1 true.

There are four differences to"naturally" spawned wrecks: 1) monster spawns won't work, 2) no corpses, 3) rooms are not randomly filled with water, and 4) the wrecks are placed at the mouse cursor and not automatically positioned. All the wrecks are static, which means they don't fall and cannot be moved by physics. All the wrecks just stay magically where they were spawned. The reason why we decided to make them static was partially because of performance and partially to ensure that they don't accidentally go where they are not intended to be and block the main route for example.
